### PR TITLE
Square Shield

### PR DESCRIPTION
**Pull request by Geevies**
* Replaced the circular buckler sprite with a more rectangular variant. Also renamed it to selfmade shield.

Comparison:
image

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/game/objects/items/weapons/improvised_components.dm [hunk 1]
```diff
@@ -118,8 +118,8 @@
 /obj/item/material/woodenshield
 	name = "shield donut"
 	desc = "A wooden disc. Unusable as a shield without metal. Don't eat this."
```
Comment (KasparoVy) [inline code/game/objects/items/weapons/improvised_components.dm:120]: This ain't a disc anymore, is it?

Comment (Geevies) [reply code/game/objects/items/weapons/improvised_components.dm:120]: it is, but muuuuch smaller

Comment (KasparoVy) [reply code/game/objects/items/weapons/improvised_components.dm:120]: Got it, cheers

**Pull request merged**